app/controllers/styled_forecasts_controller.rb

```diff
@@ -1,7 +1,7 @@
-# :nocov:
 class StyledForecastsController < ApplicationController
   layout "tailwind_layout"
   def show
+    @forecasts = CercApiClient
+      .forecasts_for(params.has_key?("zone") ? params["zone"] : "Southwark")
```

Comment:
This conditional can be simplified to `params.fetch("zone", "Southwark")`

See https://ruby-doc.org/core-2.5.1/Hash.html#method-i-fetch